meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb

```diff
@@ -45,7 +45,7 @@ RDEPENDS:${PN}-tools-dts = " \
     dasharo-deploy \
     dasharo-hcl-report \
     dts \
-    system76-ectool \
+    dasharo-ectool \
```

Comment:
@mkopec Does it work? I would guess the system76-ectool_0.3.8 must be renamed here as well.

Reply:
No, I'm working on fixing these issues and will submit a PR soon